sssd_test_framework/markers.py

```diff
@@ -67,7 +77,13 @@ def pytest_runtest_setup(item: pytest.Item) -> None:
             # @pytest.mark.builtwith(client="passkey", ipa="passkey") ->
             # -> check if "passkey" is supported by both client and ipa
             requirements = dict(mark.kwargs)
+        elif len(mark.args) == 1 and mark.args:
+            # @pytest.mark.builtwith(KnownTopology.IPA, ipa="passkey") ->
+            # -> check if "passkey" is supported by ipa
+            for iterargs in mark.args:
+                topology = iterargs
         else:
             raise ValueError(f"{item.nodeid}::{item.originalname}: invalid arguments for @pytest.mark.builtwith")
 
-        item.add_marker(pytest.mark.require(partial(builtwith, item=item, requirements=requirements)))
+        item.add_marker(pytest.mark.require(partial(
```

Comment:
```python
if not topology:
    topology = [None]

for t in topology:
   iteam.add_marker(...)
```

Reply:
Well, not exactly the same but I implemented something similar, so I'm closing this one also